src/App.js

```diff
@@ -1,17 +1,9 @@
-import Typography from '@material-ui/core/Typography';
-import './App.css';
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Please, consider using class-based for App.js component in place function. In the next steps, you'll see why this is important. 🐶

Reply:
Thank you, I missed that one.